src/styles/main.scss

```diff
@@ -1,3 +1,215 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
You should add fonts as a link tag in html in tag head
https://prnt.sc/fZFZCsPhi2qK

Reply:
@iryna-lelyk not necessarily